extensions/mongoProcessor/MongoQueueProcessor.js

```diff
@@ -211,21 +284,47 @@ class MongoQueueProcessor {
             isNFS: null,
         };
 
-        entry.setReplicationInfo(replicationInfo);
-    }
+        // TODO: refactor based off cloudserver getReplicationInfo
```

Comment:
Would it be possible to extract this into an Arsenal util so that a single place can manage this logic? Seeing this closely resemble [_getReplicationInfo](https://github.com/scality/cloudserver/blob/development/8.1/lib/api/apiUtils/object/getReplicationInfo.js#L33). Is that what's meant by the comment?

Reply:
Yup, the todo was added to revisit this. I wasn't sure how to go about the next steps for setting content, so I decided to just borrow the logic from cloudserver.
But I think it'll be ok, will work on refactoring